src/calculation/Calculator.ts

```diff
@@ -564,7 +564,15 @@ export async function calculateDataRequirements(
   // Extract the library ELM, and the id of the root library, from the measure bundle
   const { cqls, rootLibIdentifier, elmJSONs } = MeasureBundleHelpers.extractLibrariesFromMeasureBundle(measureBundle);
   const effectivePeriod = MeasureBundleHelpers.extractMeasureFromBundle(measureBundle).effectivePeriod;
-  return DataRequirementHelpers.getDataRequirements(cqls, rootLibIdentifier, elmJSONs, options, effectivePeriod);
+  const dataRequirements = await DataRequirementHelpers.getDataRequirements(
+    cqls,
+    rootLibIdentifier,
+    elmJSONs,
+    options,
+    effectivePeriod
+  );
+  dataRequirements.results.relatedArtifact = DataRequirementHelpers.getFlattenedRelatedArtifacts(measureBundle);
```

Comment:
I just realized that it might be useful to also include this on `libraryDataRequirements` (different API function).

In that case, there won't be a Measure resource (the input is assumed to be a bundle of libraries), so it might require a slight re-work. Could probably just have the helper function take in a boolean for `shouldAddMeasure` and only do the logic for extracting it and adding it to the bundle when you're doing regular data requirements?